mucommander-os-macos/src/main/java/com/mucommander/desktop/macos/OSXDesktopAdapter.java

```diff
@@ -242,4 +243,40 @@ private static String getMacOsUserShell() {
         }
         return result;
     }
+
+    @Override
+    public boolean setIconBadgeNumber(int number) {
+        // Hmm, Taskbar.isTaskbarSupported returns false on my macOS 10.15.7 and java 14.0.2
```

Comment:
as long as the dmg we publish are supported on those older versions that's ok - it would be interesting to see if 1.3.0 with Java 20 would work on macOS 10.15.7

Reply:
so, with Java 20 `Taskbar.isTaskbarSupported` returns true and it works with both badges and progressbar (but with workaround - just pushed a commit to address that, apparently a bug in AWT that exists up to 21).